### PR TITLE
chore(ndm_charts): add notes and readme for NDM charts

### PR DESCRIPTION
**Pull request by sonasingh46**
The PR adds NOTES.txt for helm charts and adds a Readme that describes the values. yaml keys
Signed-off-by: Ashutosh Kumar <ashutosh.kumar@mayadata.io>

### CODE REVIEW

File: deploy/helm/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# OpenEBS Node Disk Manager Helm Chart
+
+[OpenEBS Node Disk Manager](https://github.com/openebs/node-disk-manager) aims to make it easy to manage the disks 
+attached to the node on a Kubernetes cluster.
+Please visit the [link](https://github.com/openebs/node-disk-manager) to know more about node disk manager.
```
Comment (kmova) [inline deploy/helm/README.md:5]: This shouldn't be there / redundant.. we are already in this repo.

File: deploy/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+This chart bootstraps OpenEBS NDM deployment on a [Kubernetes](http://kubernetes.io) cluster using the 
+[Helm](https://helm.sh) package manager.
+
+## Installation
+
+You can run OpenEBS NDM on any Kubernetes 1.13+ cluster in a matter of seconds.
+
+Please visit the [link](https://openebs.github.io/node-disk-manager/) for install instructions via helm3.
+## Configuration
```
Comment (akhilerm) [inline deploy/helm/README.md:17]:
```suggestion

## Configuration
```

File: deploy/helm/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+The following table lists the configurable parameters of the OpenEBS chart and their default values.
```
Comment (akhilerm) [inline deploy/helm/README.md:19]:
```suggestion
The following table lists the configurable parameters of the OpenEBS NDM chart and their default values.
```

**Review by akhilerm — DISMISSED** [review]
A few formatting related comments, everything else looks good.

File: deploy/helm/README.md (hunk 1 above)
Comment (sonasingh46) [reply deploy/helm/README.md]: done.

File: deploy/helm/README.md (hunk 2 above)
Comment (sonasingh46) [reply deploy/helm/README.md]: done.

File: deploy/helm/README.md (hunk 3 above)
Comment (sonasingh46) [reply deploy/helm/README.md]: done.

File: deploy/helm/charts/values.yaml [hunk 4]
```diff
@@ -63,6 +63,8 @@ ndmOperator:
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
     tag: 0.9.1
+  podLabels:
+  #  name: openebs-ndm-operator
```
Comment (akhilerm) [inline deploy/helm/charts/values.yaml:67]: Why was this label added?

Comment (kmova) [reply deploy/helm/charts/values.yaml:67]: also should it be provided with empty value like below: `{}`

Comment (sonasingh46) [reply deploy/helm/charts/values.yaml]: I have propagated the name label

**Review by akhilerm — APPROVED** [review]
lgtm

**Review by kmova — APPROVED** [review]

**Pull request merged**